server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/VacationIntegrationTest.java

```diff
@@ -376,7 +376,7 @@ private void assertOneMessageReceived(AccessToken recipientToken, String mailbox
             .body(SECOND_ARGUMENTS + ".list[0].to.email[0]", equalTo(expectedTo));
     }
 
-    private boolean assertNotYetImplemented(AccessToken recipientToken, String mailboxId) {
+    private boolean assertOneHTMLMessageReceived(AccessToken recipientToken, String mailboxId, String expectedFrom, String expectedTo) {
```

Comment:
you don't check that you got an HTML message, you just check that the textBody is null.